### PR TITLE
Refactor the docuementation of BoxContainer and BoxShape classes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: doc/classes/BoxContainer.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="BoxContainer" inherits="Container" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Base class for Box containers.
+		Base class for box containers.
 	</brief_description>
 	<description>
-		Base class for Box containers. It arranges children controls vertically or horizontally, and rearranges them automatically when their minimum size changes.
+		Arranges children controls vertically or horizontally, and rearranges the controls automatically when their minimum size changes.
```
Comment (cbscribe) [inline doc/classes/BoxContainer.xml]: "child controls", not "children controls"

File: doc/classes/BoxShape.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		Box shape resource.
 	</brief_description>
 	<description>
-		Box shape resource, which can be set into a [PhysicsBody] or area.
+		Can be a child of a [PhysicsBody], [Area], or [Area2D].
```
Comment (cbscribe) [inline doc/classes/BoxShape.xml]: This is a 3D shape, I don't think it can be a child of Area2D.

Comment (deliciousfudge) [reply doc/classes/BoxShape.xml]: Just checked through the source code again and you're right, the 3D collision shape includes this class but the 2D one doesn't. Will correct.

File: doc/classes/BoxContainer.xml [hunk 3]
```diff
@@ -17,8 +17,7 @@
 			<argument index="0" name="begin" type="bool">
 			</argument>
 			<description>
-				Add a control to the box as a spacer.
-				If [i]begin[/i] is true the spacer control will be inserted in front of other children.
+				Adds a control to the box as a spacer. If [code]true[/code], [i]begin[/i] will insert the spacer control in front of other children.
```
Comment (Noshyaar) [inline doc/classes/BoxContainer.xml]: Please use [code] for every parameters name and values such as true,false.

**Pull request merged**